modules/acm_sku/src/StockManager.php

```diff
@@ -0,0 +1,446 @@
+<?php
+
+namespace Drupal\acm_sku;
+
+use Drupal\acm\Connector\APIWrapperInterface;
+use Drupal\acm\I18nHelper;
+use Drupal\acm_sku\Entity\SKUInterface;
+use Drupal\acm_sku\Entity\SKU;
+use Drupal\acm_sku_stock\Event\StockUpdatedEvent;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Database\Connection;
+use Drupal\Core\Lock\LockBackendInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * Class StockManager.
+ *
+ * @package Drupal\acm_sku
+ */
+class StockManager {
```

Comment:
Why `StockManager` isn't in `acm_sku_stock` submodule? I think it should be contained in that one. Also it should be able to use the rest of suite without `acm_sku_stock` submodule.

Reply:
I was facing issues because of dependencies of it as it is new module introduced now. I looked at changes done in acm modules and followed the approach of ProductManager which is in acm_sku even though new module acm_product was created